Services/QuotationService.php

```diff
@@ -102,6 +99,21 @@ public function calculateQuotationByPostId( $postId ) {
 		return ( new OrderQuotationService() )->saveQuotation( $postId, $quotations );
 	}
 
+	public function getPayload($postId, $products = array())
+	{
+		if (empty($products)) {
+			$productService = new OrdersProductsService();
+			$products = $productService->getProductsOrder($postId);
```

Comment:
Só pra não confundir na leitura, o que acha de:
```suggestion
			$orderProductService = new OrdersProductsService();
			$products = $orderProductService->getProductsOrder($postId);
```